src/networks.json

```diff
@@ -1196,6 +1196,22 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2400": {
+    "key": "2400",
+    "name": "TCG Verse Mainnet",
+    "shortName": "TCGV",
+    "chainId": 2400,
+    "network": "mainnet",
+    "multicall": "0xceC65DEE0b5012F1b7321b2647681F997c7204FC",
```

Comment:
Contract code here https://explorer.tcgverse.xyz/address/0xceC65DEE0b5012F1b7321b2647681F997c7204FC/contracts should be verified and published.

Reply:
@ChaituVR The contract is verified. Sorry, line breaks are shown weirdly, but the contract is correct.